arcgis-ios-sdk-samples/Scenes/View point cloud data offline/README.md

```diff
@@ -2,27 +2,31 @@
 
 Display local 3D point cloud data.
 
-![Screenshot](image1.png)
+![View point cloud data offline sample](view-point-cloud-data.png)
 
 ## Use case
 
 Point clouds are often used to visualize massive sets of sensor data such as lidar. The point locations indicate where the sensor data was measured spatially, and the color or size of the points indicate the measured/derived value of the sensor reading. In the case of lidar, the color of the visualized point could be the color of the reflected light, so that the point cloud forms a true color 3D image of the area.
```

Comment:
```suggestion
Point clouds are often used to visualize massive sets of sensor data such as LiDAR. The point locations indicate where the sensor data was measured spatially, and the color or size of the points indicate the measured/derived value of the sensor reading. In the case of LiDAR, the color of the visualized point could be the color of the reflected light, so that the point cloud forms a true color 3D image of the area.
```